README.md

```diff
@@ -1,92 +1,119 @@
+  - Names columns for consistency across all packages and for easier analysis
     * Boolean fields are prefixed with `is_` or `has_`
     * Timestamps are appended with `_at`
     * ID primary keys are prefixed with the name of the table.  For example, the user table's ID column is renamed user_id.
     * Foreign keys include the table that they refer to. For example, the project table's owner ID column is renamed owner_user_id.
-
-## Installation Instructions
-Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
-
-Include in your `packages.yml`
-
+  - Removes any rows that are soft-deleted
+  - Adds freshness tests to source data
+  - Adds column-level testing where applicable. For example, all primary keys are tested for uniqueness and non-null values.
+- Generates a comprehensive data dictionary of your Asana data through the [dbt docs site](https://fivetran.github.io/dbt_asana_source/).
+- These tables are designed to work simultaneously with our [Asana transformation package](https://github.com/fivetran/dbt_asana).
+
+# 🎯 How do I use the dbt package?
+## Step 1: Prerequisites
+To use this dbt package, you must have the following:
+
+- At least one Fivetran Asana connector syncing data into your destination.
+- A **BigQuery**, **Snowflake**, **Redshift**, or **PostgreSQL** destination.
+
+## Step 2: Install the package
+Include the following asana_source package version in your `packages.yml` file.
+> TIP: Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
 ```yaml
 packages:
   - package: fivetran/asana_source
     version: [">=0.5.0", "<0.6.0"]
 ```
 
-## Configuration
-By default, this package will run using your target database and the `asana` schema. If this is not where your Asana data is (perhaps your Asana schema is `asana_fivetran`), add the following configuration to your `dbt_project.yml` file:
+## Step 3: Define database and schema variables
+By default, this package runs using your destination and the `asana` schema. If this is not where your Asana data is (for example, if your Asana schema is named `asana_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
 config-version: 2
 
 vars:
   asana_source:
     asana_database: your_database_name
     asana_schema: your_schema_name 
 ```
+## (Optional) Step 4: Additional configurations
+<details><summary>Expand for configurations</summary>
 
-To add additional columns to tasks use the pass-through column variable.  This is useful for adding custom fields not already included in this package.
+### Passing Through Additional Metrics 
```

Comment:
Same as the other comments on passthrough metrics. These fields are not metrics that customers are passing through, but rather normal fields. Can we update this to be `Columns` instead of `Metrics`.

Reply:
updated